tests/components/radarr/__init__.py

```diff
@@ -90,13 +95,9 @@ def mock_connection(
         headers={"Content-Type": CONTENT_TYPE_JSON},
     )
 
-    movie_fixture = "movie"
-    if single_return:
-        movie_fixture = f"single-{movie_fixture}"
```

Comment:
This was not needed. We always get a list when calling the endpoint with no arguments